collectors/jiraffe/convertors.py

```diff
@@ -0,0 +1,180 @@
+"""
+transform Jira issue into OSIDB tracker model
+"""
+import uuid
+from functools import cached_property
+
+from django.conf import settings
+
+from osidb.core import generate_acls, set_user_acls
+from osidb.models import Tracker
+
+from ..utils import (
+    tracker_parse_update_stream_component,
+    tracker_summary2module_component,
+)
+
+
+class TrackerConvertor:
```

Comment:
kinda weird that this common base class is in Jiraffe

Reply:
I was thinking whether **bzimport** or **Jiraffe** but the other way there was a cyclic import so this way it was simpler. We could find some common place between the collectors.

Reply:
The reason is to prevent a cycle. The **bzimport** collector uses the **Jiraffe** and not the opposite. We could maybe create some base module for the trackers but as they only share a part of conversion logic I am not sure it is worth.